couchbase/tests/common.py

```diff
@@ -188,9 +188,6 @@
     'cpu_sys_percent',
     'cpu_user_percent',
     'memory_total',
-    'request_per_sec_15min',
```

Comment:
I was worried that this would also affect unit tests, but then I saw that we're not asserting any metrics in couchbase 🙈